protocol/v2/ssv/validator/committee_queue.go

```diff
@@ -30,24 +30,28 @@ func (c *Committee) HandleMessage(logger *zap.Logger, msg *queue.SSVMessage) {
 		return
 	}
 
-	c.mtx.RLock() // read v.Queues
-	q, ok := c.Queues[slot]
-	c.mtx.RUnlock()
-	if !ok {
-		q = queueContainer{
-			Q: queue.WithMetrics(queue.New(1000), nil), // TODO alan: get queue opts from options
-			queueState: &queue.State{
-				HasRunningInstance: false,
-				Height:             specqbft.Height(slot),
-				Slot:               slot,
-				//Quorum:             options.SSVShare.Share,// TODO
-			},
-		}
+	prepareQueue := func() queueContainer {
```

Comment:
can we extract this into a new method for readability?

Reply:
Yep, addressed in https://github.com/ssvlabs/ssv/pull/1784/commits/483c2ed41d9b789251d57e9d6ce4f3275e3f02c1